boofuzz/sessions.py

```diff
@@ -1063,7 +1066,17 @@ def transmit_fuzz(self, sock, node, edge, callback_data):
     def build_webapp_thread(self, port=constants.DEFAULT_WEB_UI_PORT):
         app.session = self
         http_server = HTTPServer(WSGIContainer(app))
-        http_server.listen(port)
+        while True:
+            try:
+                http_server.listen(port)
+            except socket.error as exc:
+                # Only handle "Address already in use"
+                if exc.args[0] != 98:
```

Comment:
For code readability I'd recommend using the `errno` module: https://docs.python.org/2/library/errno.html#errno.EADDRINUSE

Something like:
```python
            if exc.errno != errno.EADDRINUSE:
```
and of course `import errno` at the top. And then the comment can be removed as it is implied in the code.